### PR TITLE
Enabled 114 support

### PR DESCRIPTION
**Pull request by HimajaDhanyamraju2**

### CODE REVIEW

File: registration-default.properties [hunk 1]
```diff
@@ -475,12 +475,18 @@ mosip.kernel.otp.expiry-time=${mosip.kernel.otp.expiry-time}
 
 ## backward compatibility fields	
 mosip.registration.uin.update.configured.fields=name,age,gender,address,phone,email,parentOrGuardianDetails,foreigner,biometrics,cnieNumber
-mosip.primary-language=${mosip.primary-language}	
-mosip.secondary-language=${mosip.secondary-language}	
+mosip.primary-language=eng
+mosip.secondary-language=ara
```
Comment (vishwa-vyom) [inline registration-default.properties:479]: why the existing value is ${mosip.primary-language}, is this suppose to be configured in config map of config server ?

File: registration-default.properties [hunk 2]
```diff
@@ -475,12 +475,18 @@ mosip.kernel.otp.expiry-time=${mosip.kernel.otp.expiry-time}
...
 mosip.registration.ideal_time=900	
 mosip.kernel.applicant.type.age.limit=0	
 mosip.registration.max_age=150
 
+#Temporary addition to support upgrade
+client.upgrade.server.url=https://regclient.upgrade2.mosip.net
+
+# Support Reg.Client upgrade
+mosip.registration.verion.upgrade.version-mappings={ "1.1.4":{ "dbVersion":"1.1.4", "releaseOrder":1}, "1.1.5":{ "dbVersion":"1.1.5", "releaseOrder":2}, "1.1.5.5":{ "dbVersion":"1.1.5.5", "releaseOrder":3}, "1.2.0.1-SNAPSHOT":{ "dbVersion":"1.2.0.1", "releaseOrder":4} }
+
```
Comment (vishwa-vyom) [inline registration-default.properties]: Last time when we were testing upgrade, how did we missed to add it in 1.2.0.1-v3 config branch ?

**Review by vishwa-vyom — CHANGES_REQUESTED** [review]

File: registration-default.properties (hunk 2 above)
Comment (HimajaDhanyamraju2) [reply registration-default.properties]: This is version-mappings configuration, and should be configured based on whatever version we are upgrading from. Even if this property doesn't exist, we have a default-version-mappings key added in registration-client spring.properties, based on which, the upgrade will progress.

File: registration-default.properties (hunk 1 above)
Comment (HimajaDhanyamraju2) [reply registration-default.properties:479]: The place-holder mentioned, is not present in application-default.properties to fill in the values. If you suggest to add them there, will add those two properties in application-default.properties @vishwa-vyom

**Review by vishwa-vyom — APPROVED** [review]

**Pull request merged**